src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -75,6 +81,8 @@ const defaultProps = {
     isAddPaymentMenuActive: false,
     shouldShowAddPaymentMethodButton: true,
     filterType: '',
+    actionPaymentMethodType: '',
+    activePaymentMethodID: NaN,
```

Comment:
Since we're setting an empty string as default value [here](https://github.com/Expensify/App/blob/277512661d4be6d0f5a93d72a970c2cfd75c9a4c/src/pages/settings/Payments/PaymentsPage.js#L265-L267) I think we should also set an empty string by default in this line, no?
 
```suggestion
    activePaymentMethodID: '',
```

Reply:
Yeah, I used number type for the prop so I set it to NaN. I will correct.